common/netmap.go

```diff
@@ -12,6 +14,11 @@ import (
 // a successive call is not guaranteed.
 // Caller must have `NewEpoch` method with a single numeric argument.
 func SubscribeForNewEpoch() {
+	callingHash := runtime.GetExecutingScriptHash()
+	if !management.HasMethod(callingHash, "newEpoch", 1) {
+		panic(std.Base58Encode(callingHash) + " contract does not have `newEpoch(epoch)` method")
```

Comment:
`std.Base58Encode` is not the best choice here, it's not an address and it's not a hash representation. You can try https://pkg.go.dev/github.com/nspcc-dev/neo-go/pkg/interop@v0.0.0-20231123162724-195bb742ca16/native/std#Itoa or https://github.com/nspcc-dev/neo-go/blob/195bb742ca16/pkg/interop/lib/address/address.go#L28.

Reply:
ok, `address.FromHash160`